tools/markdown-magic/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "@fluid-tools/markdown-magic",
+	"version": "0.1.0",
+	"private": true,
+	"description": "Contains shared utilities for Markdown content generation and embedding using markdown-magic.",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "tools/markdown-magic"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"sideEffects": false,
+	"bin": {
+		"markdown-magic": "bin/markdown-magic"
+	},
+	"main": "src/index.js",
+	"scripts": {
+		"build": "npm run build:docs",
+		"build:docs": "node src/index.js --files README.md",
+		"format": "npm run prettier:fix",
+		"lint": "npm run prettier && npm run eslint",
+		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
+		"prettier": "prettier --check .",
+		"prettier:fix": "prettier --write .",
+		"test": "node src/index.js --files test/**/*.md"
+	},
+	"dependencies": {
+		"@fluidframework/build-common": "^1.1.0",
+		"@fluidframework/eslint-config-fluid": "^2.0.0",
+		"@tylerbu/markdown-magic": "^2.4.0-tylerbu-1",
```

Comment:
We should get rid of this eventually. Not a blocker for this PR.

Reply:
I was actually going to ask you about that. Almost seems as though we could just move that logic into this private package instead.